source/common/router/router_ratelimit.cc

```diff
@@ -257,18 +308,44 @@ QueryParameterValueMatchAction::QueryParameterValueMatchAction(
     Server::Configuration::CommonFactoryContext& context)
     : descriptor_value_(action.descriptor_value()),
       descriptor_key_(!action.descriptor_key().empty() ? action.descriptor_key() : "query_match"),
+      default_value_(action.default_value()),
+      expect_match_(PROTOBUF_GET_WRAPPED_OR_DEFAULT(action, expect_match, true)),
+      action_query_parameters_(
+          buildQueryParameterMatcherVector(action.query_parameters(), context)),
+      descriptor_formatter_(nullptr) {}
```

Comment:
We needn't to initialize the default_value_ and descriptor_formatter_ in this constructor.